packages/lesswrong/lib/sql/PgCollection.ts

```diff
@@ -97,7 +97,7 @@ class PgCollection<T extends DbObject> extends MongoCollection<T> {
       count: async () => {
         const select = new SelectQuery(this.getTable(), selector, options, {count: true});
         const result = await this.executeQuery(select, {selector, options});
-        return result?.[0].count ?? 0;
+        return parseInt(result?.[0].count ?? 0);
```

Comment:
Count was being returned as a string which caused a bug when checking `if (await Tags.find({parentTagId: oldDocument._id}).count())` (it evaluated to true when it should be false)